runtime/src/main/scala/akka/grpc/Trailers.scala

```diff
@@ -36,4 +36,9 @@ class Trailers(val status: Status, val metadata: Metadata) {
 object Trailers {
   def apply(status: Status): Trailers = new Trailers(status)
   def apply(status: Status, metadata: Metadata): Trailers = new Trailers(status, metadata)
+
+  def apply(code: com.google.rpc.Code, message: String, details: Seq[scalapb.GeneratedMessage]): Trailers = {
+    val ex = GrpcServiceException(code, message, details)
+    Trailers(ex.status, ex.metadata)
+  }
```

Comment:
Since this is common API for Java and Scala we should have one that accepts a Java collection as well, and transform/delegate. Usually in Akka land we call those `create` but just another `apply` overload converting the java collection using `.toScala.toSeq` would suffice.

I think it's fine to delegate to a throw-away GrpcServiceException for now, once we need it in a third place we can revisit extracting some utility.